lib/jazzy/themes/apple/assets/css/jazzy.css.scss

```diff
@@ -529,3 +529,70 @@ html.dash {
     position: static;
   }
 }
+
+// ===========================================================================
+//
+//  Search
+//
+// ===========================================================================
+form[role=search] {
+  float: right;
+
+  input {
+    font: Helvetica, freesans, Arial, sans-serif;
+    margin-top: 3px;
+    margin-left: 16px;
+    font-size: 13px;
+    line-height: 26px;
```

Comment:
The design has a very skinny header that makes it tough but I think it needs more spacing around the entryfield to avoid the very thin black borders, something like:
```css
margin-top: 6px;
line-height: 20px;
```
![Screenshot 2020-06-07 at 09 48 04](https://user-images.githubusercontent.com/26768470/83964779-2b703480-a8a7-11ea-8a49-41f2849fa040.png)
![Screenshot 2020-06-07 at 09 49 08](https://user-images.githubusercontent.com/26768470/83964780-2ca16180-a8a7-11ea-85d2-1871c4f75853.png)
 
What is the `margin-left` for?  It looks to be  making the dropdown not align with the entryfield.